AIDevGallery/Samples/WCRAPIs/BackgroundRemover.xaml

```diff
@@ -9,100 +9,72 @@
     xmlns:samples="using:AIDevGallery.Samples"
     xmlns:shared="using:AIDevGallery.Samples.SharedCode"
     mc:Ignorable="d">
-    <Grid ColumnSpacing="24" RowSpacing="12">
+    <Grid RowSpacing="12">
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="*" />
-            <ColumnDefinition Width="*" />
-        </Grid.ColumnDefinitions>
+        <Viewbox 
+            x:Name="ImageViewbox" 
+            Grid.Row="0"
+            Stretch="Uniform"
+            MaxWidth="600"
+            MaxHeight="400">
+            <Border 
+                BorderThickness="3" 
+                BorderBrush="{ThemeResource AccentAAFillColorDefaultBrush}"
+                CornerRadius="3">
+                <Grid>
+                    <Image 
+                        x:Name="CanvasImage"
+                        PointerPressed="CanvasImage_PointerPressed"/>
+                    <Canvas 
+                        x:Name="PointsCanvas" 
+                        IsHitTestVisible="False"/>
+                </Grid>
+            </Border>
+        </Viewbox>
         <StackPanel
-            Grid.ColumnSpan="2"
             Orientation="Horizontal"
-            Spacing="36">
+            HorizontalAlignment="Center"
+            Spacing="8"
+            Grid.Row="1">
+            <Button
+                x:Name="RevertButton"
+                Content="Revert"
+                Style="{ThemeResource AccentButtonStyle}"
+                Click="RevertButton_Click"
+                Visibility="Collapsed"/>
+            <Button 
+                x:Name="RemoveBackgroundButton"
+                Content="Remove Background"
+                Style="{ThemeResource AccentButtonStyle}"
+                Click="RemoveBackground_Click"/>
+            <Button
+                x:Name="ClearSelectionButton"
+                Content="Clear Selected Points"
+                Click="CleanSelection_Click"/>
             <DropDownButton AutomationProperties.Name="Select image">
-                <StackPanel Orientation="Horizontal" Spacing="8">
-                    <FontIcon FontSize="16" Glyph="&#xEE71;" />
-                    <TextBlock Text="Select image" />
-                </StackPanel>
+                <TextBlock Text="Select Image" />
```

Comment:
No need for the `AutomationProperties.Name` or `TextBlock` if we move this to the dropdownbutton `Content`